src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedObject = {};
+
+  const splited = sourceString.split(';');
+
+  splited.forEach((split) => {
+    if (split.trim()) {
+      const [property, value] = split.split(':');
```

Comment:
Be cautious with the 'split' method. If the property value contains a colon, it will split the value incorrectly. While it's not required to handle multiple colons, it would be a good practice to account for such cases by using a limit on the 'split' method or a different approach to ensure correct parsing.